src/Symfony/Component/ErrorHandler/Exception/FlattenException.php

```diff
@@ -359,7 +359,6 @@ private function flattenArgs(array $args, int $level = 0, int &$count = 0): arra
                 return ['array', '*SKIPPED over 10000 entries*'];
             }
             if ($value instanceof \__PHP_Incomplete_Class) {
```

Comment:
Should be removed too?

Reply:
Nope, this one is still needed to get the correct class name.